src/mono/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.Mono.cs

```diff
@@ -236,7 +236,7 @@ public override bool ReflectionOnly
             get { return base.ReflectionOnly; }
         }
 
-        [RequiresDynamicCode("Generating new code at runtime is not supported with native AOT.")]
+        [RequiresDynamicCode("Creating a DynamicMethod requires dynamic code.")]
```

Comment:
These two aren't for creating a DynamicMethod:

```suggestion
        [RequiresDynamicCode("Defining a dynamic assembly requires dynamic code.")]
```